packages/ai/core/types/transcription-model-response-metadata.ts

```diff
@@ -0,0 +1,16 @@
+export type TranscriptionModelResponseMetadata = {
+  /**
+Timestamp for the start of the generated response.
+   */
+  timestamp: Date;
+
+  /**
+The ID of the response model that was used to generate the response.
+   */
+  modelId: string;
+
+  /**
+Response headers.
+   */
+  headers?: Record<string, string>;
+};
```

Comment:
wonder if we should reuse the more general type in the `ai` lib (not in the spec)

Reply:
Sure - this is copied from `generateImage` but let's catch up and see if there's a better way to handle it.